### ISSUE TITLE
latest uf2 file location

### ISSUE THREAD

**Issue by Tom-ak2b** [dawsonjon/PicoRX#153]
Can someone please tell me where to find the latest precompiled file location.

**Comment by letonton** [comment]
Hello
there's a link in git page, section Getting the code

**Comment by dawsonjon** [comment]
Yes, that's right, the link takes you to the "actions" tab on the GitHub page, under there, select the latest build, then you will find 3 zip files containing the .uf2 for the various platforms. I would like to make a link that takes you straight to the latest zip file, but don't know how🤔
